src/nativeComponents.ts

```diff
@@ -64,13 +64,34 @@ function readMetadata(elfPath: string) {
   const appIDData = findSection('appuuid');
   checkBufferLength(appIDData, 16, 'App UUID');
 
+  const platformJSON = findSection('appplatform').toString();
+
+  let platform;
+  try {
+    platform = JSON.parse(platformJSON);
+  } catch (ex) {
+    throw new PluginError(
+      PLUGIN_NAME,
+      'Platform specification JSON is invalid',
```

Comment:
Include the section name in the error message so it's more actionable to people who aren't familiar with the details of how that's stored in the ELF.
```suggestion
      `Could not parse platform specification in .appplatform section: ${ex.message}`,
```